python/cuml/cuml/internals/api_decorators.py

```diff
@@ -82,6 +85,13 @@ def _get_value(args, kwargs, name, index, default_value):
                 )
 
 
+def support_array_like(value):
+    if isinstance(value, (list, tuple)):
```

Comment:
This function is only called in one place (besides tests), and is small enough that it would make more sense to me to inline it. The mental cost of the indirection here isn't worth it IMO (I'd agree with my old colleague here https://matthewrocklin.com/blog/work/2019/06/23/avoid-indirection) and I'd prefer we remove the helper. The call within `CumlArray` inlines the method already anyway.